webextension/background.js

```diff
@@ -0,0 +1,9 @@
+try {
+  browser.cloudstorage.init(browser.runtime.getURL("./skin/clouddownloads.css")).then(
```

Comment:
Looking forward to review this PR after the small-study utils example has been implemented: https://github.com/mozilla/shield-studies-addon-utils/blob/develop/examples/small-study/src/background.js :)

Reply:
True. Until then, there is an example of this in the tests: https://github.com/mozilla/shield-studies-addon-utils/blob/develop/examples/test-addon/test/functional/browser.study.js#L121

Reply:
Thanks, will browser.study.sendTelemetry call available from privileged code? my understanding from last synchup with Gregg, it's not and will be done via events. I might still be missing something here

Reply:
You are correct. That will need to be done via events. See the example in the updated template for an example of this: https://github.com/mozilla/shield-studies-addon-template/pull/55

Reply:
Thank you!